doc/tutorials/content/sources/template_alignment/template_alignment.cpp

```diff
@@ -29,8 +29,6 @@ class FeatureCloud
       feature_radius_ (0.02f)
     {}
 
-    ~FeatureCloud () {}
```

Comment:
Might need to verify that the line numbers still match in the tutorial? Seems to be off already.

Reply:
I think if you delete lines 32 and 33 in this file and replace lines 143 and 144 with empty lines instead of deleting them completely, the line number in template_alignment.rst will again (mostly) fit together with the lines in template_alignment.cpp

Reply:
Fixed. This MR could be squashed than.